src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -342,6 +342,10 @@ public function has($id)
      */
     public function get($id)
     {
+        if ('request' === $id) {
+            trigger_error('The "request" service is deprecated and will be removed in 3.0. Add a typehint for Symfony\\Component\\HttpFoundation\\Request to your controller parameters to retrieve the request instead.', E_USER_DEPRECATED);
+        }
+        
```

Comment:
for other deprecations, the deprecation is on the class itself, not on the fact to use them as services. So there is already deprecation warnings because of the use of the class